### PR TITLE
Homework 1

### PR DESCRIPTION
**Pull request by prachi-kashyap**
Fizzbuzz Program - First Assignment

### CODE REVIEW

**Review by ethanSE — COMMENTED** [review]
The code works and looks good. She's asking us to use the Airbnb style guide for all JS submitted for the class. I'm not sure if you're using VSCode but if so here is a link to a blog post outlining setting up linting with ESLint and code formatting with Prettier according to the Airbnb style guide:
 
https://vicvijayakumar.com/blog/eslint-airbnb-style-guide-prettier

Thanks to Leshi for finding and sharing this guide

**Comment by prachi-kashyap** [comment]
Recommitted the code, after doing the whole Airbnb style setup. Thanks for the suggestion @ethanSE .

**Comment by ethanSE** [comment]
The guide I posted a link to (and also trusted myself) contains a configuration file for Prettier which removes semicolons.  However, the airbnb style guide specifically states that you should use semicolons. 

Airbnb guide on semicolons: https://github.com/airbnb/javascript#semicolons

.prettierrc
```JSON
{
    // ...
    "semi": false, // <- change to true
    // ...
}
```

**Review by ethanSE — APPROVED** [review]

File: hw1/01-fizzbuzz.js [hunk 1]
```diff
@@ -1,8 +1,17 @@
...
+
 const fizzbuzz = () => {
-  // Add your code here
-};
+    for (let number = 1; number <= 100; number += 1) {
+        let wordToPrint = number
+        if (number % 3 === 0 && number % 5 === 0) wordToPrint = 'Fizz Buzz'
+        else if (number % 3 === 0) wordToPrint = 'Fizz'
+        else if (number % 5 === 0) wordToPrint = 'Buzz'
+        console.log(wordToPrint)
```
Comment (sarvesh2910) [inline hw1/01-fizzbuzz.js]: Small change. printed words are capitalized and there is space between fizzbuzz. 
Nothing major. It is just that, had this been setup with test units or say leetcode question, program would have failed. 

Rest looks good to me.

**Review by sarvesh2910 — CHANGES_REQUESTED** [review]

File: hw1/01-fizzbuzz.js (hunk 1 above)
Comment (prachi-kashyap) [reply hw1/01-fizzbuzz.js]: Updated. I didn't see that as the major requirement, that's why wrote it like that.

**Review by sarvesh2910 — APPROVED** [review]

File: hw1/01-fizzbuzz.js (hunk 1 above)
Comment (sarvesh2910) [reply hw1/01-fizzbuzz.js]: Approved.

**Pull request merged**